src/Dofs/DofHandler.jl

```diff
@@ -12,6 +12,16 @@ Access some grid representation for the dof handler.
 """
 get_grid(dh::AbstractDofHandler)
 
+"""
+    SubDofHandler(dh::AbstractDofHandler, cellset)
+
+Create an `sdh::SubDofHandler` for the cells in `cellset`. All cells in `cellset`
+must have the same type. Fields are added to the `sdh` by using [`add!`](@ref) the 
+same way as for the standard [`DofHandler`](@ref).
+[`close!`](@ref) should be called on the parent dof handler, `dh`, after
+all `SubDofHandler`s have been created and their fields have been added. 
```

Comment:
Maybe we should add a very simple and concise example to the docstring. What do you think?

Reply:
Good idea! 
Done following @fredrikekre's `DofHandler` example!